phy/io/kwik/creator.py

```diff
@@ -297,9 +299,18 @@ def add_clustering(self,
                                        )
 
 
-def create_kwik(prm_file=None, kwik_path=None, probe=None, **kwargs):
+def create_kwik(prm_file=None, kwik_path=None, overwrite=False,
+                probe=None, **kwargs):
     prm = _read_python(prm_file) if prm_file else {}
-    sample_rate = kwargs.get('sample_rate', prm.get('sample_rate'))
+
+    if prm:
+        assert 'spikedetekt' in prm
```

Comment:
todo: we should be able to convert to .kwik without running spike detection (i.e. separate those two processes, and manually specify / detect spikes using another algorithm)